plugins/postcss-nesting/README.md

```diff
@@ -71,6 +71,73 @@ import postcssNesting from "https://cdn.jsdelivr.net/npm/postcss-nesting@10/mod.
 await postcss([postcssNesting]).process(YOUR_CSS /*, processOptions */);
 ```
 
+## Options
+
+### noIsPseudoSelector
+
+Before :
+
+```css
+#alpha,
+.beta {
+	&:hover {
+		order: 1;
+	}
+}
+
+.alpha > .beta {
+	& + & {
+		order: 2;
+	}
+}
+```
+
+After **without** the option :
+
+```js
+postcssNesting()
+```
+
+```css
+:is(#alpha,.beta):hover {
+	order: 1;
+}
+```
+
+_`.beta:hover` has specificity as if `.beta` where an id selector, matching the specification._
+
+```css
+:is(.alpha > .beta) + :is(.alpha > .beta) {
+	order: 2;
+}
+```
+
+After **with** the option :
+
+```js
+postcssNesting({
+	noIsPseudoSelector: true
+})
+```
+
+```css
+#alpha:hover, .beta:hover {
+	order: 1;
+}
+```
+
+_`.beta:hover` has specificity as if `.beta` where an class selector, conflicting with the specification._
+
+```css
+.alpha > .beta + .alpha > .beta {
+	order: 2;
+}
+```
+
+_this is a different selector than expected_
+_avoid these cases when you disable `:is()`_
+_writing the selector without nesting is advised here_
```

Comment:
I think we could do some more teaching here if we want to. This is discouraging but not actually clear, look at this

https://polypane.app/css-specificity-calculator/#selector=.alpha%20%3E%20.beta%20%2B%20.alpha%20%3E%20.beta

vs 

https://polypane.app/css-specificity-calculator/#selector=%3Ais(.alpha%20%3E%20.beta)%20%2B%20%3Ais(.alpha%20%3E%20.beta)

Which is the same (shouldn't?) 

Let me know if you want to discuss this further.

Reply:
I like that!
I added links to this site where applicable and I also split up the edge cases.

Mixing them was indeed confusing :

https://github.com/csstools/postcss-plugins/blob/postcss-nesting-less-is-pseudo--rational-curly-coated-retriever-dedd642a50/plugins/postcss-nesting/README.md#options